microsite/data/plugins/azure-resource.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Azure Resource
+author: Anurag Pachauri
+authorUrl: https://github.com/anuragpachauri
+category: Infrastructure
+description: A plugin showing Azure resource based on tag in relation to an entity in the catalog.
+documentation: https://github.com/anuragpachauri/backstageMain/tree/main/plugins/azure-resource
+npmPackageName: '@anuragpachauri/azure-resource'
```

Comment:
The repository and homepage links for this packages on NPM also 404, can you correct that, please?